civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -433,6 +433,29 @@ function civihr_employee_portal_update_7033() {
   TaxonomyHelper::createDefaultTerms();
 }
 
+/**
+ * Revert updated features components
+ * - Panel page which holds the My Details page
+ * - Output Filter to write pure HTML in views sections without filters
+ * Refresh the node files as onboarding form node was altered
+ */
+function civihr_employee_portal_update_7034() {
+  features_revert(['civihr_employee_portal_features' => ['page_manager_pages']]);
+  features_revert(['civihr_employee_portal_features' => ['filter']]);
+  civicrm_initialize();
+  drush_civihr_employee_portal_refresh_node_export_files();
+}
+
+/**
+ * Revert emergency contacts view
+ */
+function civihr_employee_portal_update_7035( ) {
+  $view = views_get_view('emergency_contacts');
```

Comment:
@mickadoo  Once a view is loaded or constructed in DB it doesn't change only because its code changes. Drupal can detect the view has changed but won't overwrite it just because the cache is cleared. I will double check this anyway. Thanks.

Reply:
This is for reference of how to revert a view when its code changes:
https://api.drupal.org/api/views/drush%21views.drush.inc/function/views_revert_view/7.x-3.x